website/src/docs/hotchocolate/v14/fetching-data/dataloader.md

```diff
@@ -113,13 +113,13 @@ It executes resolvers until the queue is empty and then triggers the data loader
 
 # Data Consistency
 
-Dataloader do not only batch calls to the database, they also cache the database response.
+DataLoader do not only batch calls to the database, they also cache the database response.
```

Comment:
Should be plural, Dataloaders

Reply:
Yeah these changes are literally what I originally suggested and I got feedback to NOT make them plural and instead use the PascalCase version. I don't have a horse in the race nor do I know what the correct usage is; I was just trying to help out and make things more consistent because this store out to me when I was reading the docs. I'm happy to go through and make changes with some overall guidance; didn't mean to take a bunch of resources with a casing change.

Reply:
Gotcha!  I was just tossing in an opinion. I'm not the one to give the official guidance.

Reply:
According to Michael, it's an invariant noun (like middleware), so there's no plural form with an `s` suffix. 🤷‍♂️

Reply:
Thanks both; so I think Michael would still need to approve since it's blocked with "changes requested"? Or maybe that's not an accurate interpretation? I'm a little rusty on this GitHub workflow.

Reply:
Michael will merge this when he gets a chance. Thanks.